src/trainer/Trainer.js

```diff
@@ -199,341 +195,189 @@ class Trainer extends Component {
+          <Input
+            id="authorName"
+            value={this.state.authorName}
+            onChange={this.handleChange('authorName')}
+          />
+        </Form.Item>
+        <Form.Item {...formItemLayout} label="Data">
+          <Select value={this.state.dataName} onChange={this.handleDataChange}>
+            {dataOptions}
+          </Select>
+        </Form.Item>
+
+        <Form.Item {...formItemLayout} label="Model">
+          <Select
+            value={this.state.modelName}
+            onChange={this.handleSelectChange('modelName')}
+          >
+            <Select.Option value={'resnet'}>ResNet</Select.Option>
+          </Select>
+        </Form.Item>
+
+        {/* TODO: Colllapse advanced options + state variables*/}
+        {/*<Form.Item {...formItemLayout} label="Split Type">*/}
+        {/*<Select*/}
+        {/*value={this.state.modelName}*/}
+        {/*onChange={this.handleChange('modelName')}*/}
+        {/*>*/}
+        {/*<Select.Option value={'train-val'}>*/}
+        {/*Training and Validation*/}
+        {/*</Select.Option>*/}
+        {/*<Select.Option value={'train-test-val'}>*/}
+        {/*Training, Test, and Validation*/}
+        {/*</Select.Option>*/}
+        {/*</Select>*/}
+        {/*</Form.Item>*/}
+
+        <Button color="primary" onClick={this.sendJobRequest}>
+          Train Model
+        </Button>
+      </Form>
+    );
+  }
+
+  /**
+   * Returns a form which can be used to view
+   * different results.
+   *
+   * The returned view is reliant on various state fields.
+   */
+  renderResultsForm() {
     const sortedAllPlots = this.state.allPlots
       .slice()
       .sort((a, b) => sortByDate(a, b, false));
-    const plotOptions = [];
-    sortedAllPlots.forEach(e => {
-      plotOptions.push(
-        <option key={e} value={e}>
-          {e}
-        </option>
-      );
-    });
 
-    // TODO(luke): Consider at another time
-    const transformSelects = [];
-    for (let i = 0; i < this.state.numTransforms; i++) {
-      let selected = '';
-      if (this.state.selectedTransforms.length > 0) {
-        selected = this.state.selectedTransforms[i];
-      }
-      transformSelects.push(
-        <FormControl style={styles.inputField}>
-          <InputLabel>Transform {i + 1}</InputLabel>
+    const plotOptions = sortedAllPlots.map(e => (
+      <Select.Option key={e} value={e}>
+        {e}
+      </Select.Option>
+    ));
+
+    return (
+      <Form>
+        <Form.Item label="Plots">
           <Select
-            native
-            disabled
-            value={selected}
-            onChange={this.handleTransformChange(i)}
+            value={this.state.selectedPlot}
+            onChange={this.handlePlotChange}
           >
-            {transformOptions}
+            {plotOptions}
           </Select>
-        </FormControl>
-      );
-    }
+        </Form.Item>
+
+        <Button href="http://104.196.51.205:5601/">Kibana</Button>
```

Comment:
Ideally we want some way for users to input their Kibana / Slack credentials, instead of hardcoding for elvo-analysis

Reply:
I agree